lib/utils.js

```diff
@@ -138,23 +125,10 @@ class Utils {
             }
         });
     }
-    /**
-     * Return true if should use 'jfrog rt c' instead of 'jfrog c'.
-     * @returns true if should use 'jfrog rt c' instead of 'jfrog c'.
-     */
-    static useOldConfig() {
-        let version = core.getInput(Utils.CLI_VERSION_ARG);
-        if (version === this.LATEST_CLI_VERSION_ARG) {
-            return false;
-        }
-        return semver.lt(version, this.NEW_CONFIG_CLI_VERSION);
-    }
 }
 exports.Utils = Utils;
 Utils.USER_AGENT = 'setup-jfrog-cli-github-action/' + require('../package.json').version;
-Utils.SERVER_TOKEN_PREFIX = /^JF_ARTIFACTORY_.*$/;
-// Since 1.45.0, 'jfrog rt c' command changed to 'jfrog c add'
-Utils.NEW_CONFIG_CLI_VERSION = '1.45.0';
+Utils.SERVER_TOKEN_PREFIX = /^JFROG_SERVER.*$/;
 Utils.CLI_VERSION_ARG = 'version';
-Utils.MIN_CLI_VERSION = '1.29.0';
+Utils.MIN_CLI_VERSION = '2.6.0';
```

Comment:
Let's have 2.12.0 as the minimal CLI version. Since people will be manually migrating to the new "v2-jf" major version, there's probably no reason to support older CLI versions.